cumulusci.yml

```diff
@@ -99,6 +99,17 @@ flows:
             3:
                 task: deploy_k12_kit_settings
 
+    config_regression:
+        steps:
+            2:
+                task: enable_course_connections
+                options:
+                    managed: True
+            3:
+                task: deploy_trial_config
```

Comment:
@jstvz does this deploy anything that the admin profile needs permissions for? if so you'll need update_admin_profile afterward.

Reply:
No, it's only deploying the admin profile itself. That said I re-ordered the steps to future-proof it.

Reply:
@jstvz are there defaults for unmanaged and managed values if they're not set?

Reply:
@jofsky: Yes, you can see them via `cci task info`.